machine/machine_test.go

```diff
@@ -176,6 +177,39 @@ func TestHalt(t *testing.T) {
 	}
 }
 
+func BenchmarkNewAndLoadLinuxWithBZimage(b *testing.B) {
```

Comment:
Could you please take a look at the Travis CI message and fix?
The message is mainly from linter.
https://app.travis-ci.com/github/bobuhiro11/gokvm/jobs/600467277